.eslintrc

```diff
@@ -151,7 +151,7 @@ rules:
   no-implicit-coercion: [1]
   no-implicit-globals: [0]
   no-implied-eval: [2]
-  no-inline-comments: [0]
+  no-inline-comments: [2]
```

Comment:
Damn, it actually was a bad idea: the usercss PR will have a lot of conflicts because of this. This is not what I was hoping for when I asked for eslint rule... Such global cosmetic changes should occur in the absence of other PRs. I think it's better to revert the rule and the changes to the unrelated code.